day_7/10_speckled_frogs.rb

```diff
@@ -0,0 +1,8 @@
+puts "How many frogs would you like to start with?"
+frog_num = gets.chomp.to_i
+
+for i in (frog_num).downto(1)
+
+  puts "#{i} speckled frogs sat on a log\neating some most delicious bugs.\nOne jumped in the pool where its nice and cool,\nthen there were #{i-1} speckled frogs.\n\n "
```

Comment:
I believe the phrase changes as the numbers go down. How would you reflect that in your code

Reply:
I would probably have a conditional statement when the current iteration is equal to 1. Is that what you were referring to?

Reply:
Yeah, I think that would accomplish it